robot/pmm/resources/pmm.py

```diff
@@ -25,10 +31,37 @@ def __init__(self, debug=False):
         self.debug = debug
         self.current_page = None
         self._session_records = []
+        self.val=0
+        self.payment_list= []
```

Comment:
u dont need these variables, I had added these in NPSP as I was using them in a some keywords

Reply:
Fixed